source/includes/openstack/_securityGroupRules.md

```diff
@@ -0,0 +1,142 @@
+ `endPort`<br/>*int*            | Maximum port that is matched by this rule             
+ `etherType`<br/>*string*       | IP type, either IPv4 or IPv6              
+ `protocol`<br/>*string*        | The IP protocol, either ICMP, TCP or UDP             
+ `ingress`<br/>*boolean*        | Direction in which the rule is applied, false implies egress            
+ `securityGroupId`<br/>*UUID*   | The ID of the parent security group             
+
+
+#### Retrieve a security group rule
+
+```shell
+curl -H "MC-Api-Key: your_api_key" \
+    "https://api.your.cloudmc/v1/services/compute-os/devel/securitygrouprules/655d3bcb-3f8a-4738-b50a-53bca43469b5"
+```
+```json
+{
+	 "data": {
+		   "id": "655d3bcb-3f8a-4738-b50a-53bca43469b5",
+		   "remoteAddress": "0.0.0.0/0",
+		   "startPort": 1,
+		   "endPort": 65535,
+		   "etherType": "IPv4",
+		   "protocol": "ICMP",
+		   "ingress": true,
+		   "securityGroupId": "f54f050b-01b2-4a73-b6e1-4e13a5566323"
+	  }
+}
+```
+
+<code>GET /services/<a href="#service-connections">:service_code</a>/<a href="#environments">:environment_name</a>/securitygrouprules/:id</code>
+
+Retrieve information about a security group rule.
+
+Attribute                      | Description                          
+---------------------------    | ------------------------------------
+`id`<br/>*UUID*                | The security group rule's id         
+`remoteAddress`<br/>*string*   | Remote IPs or CIDRs associated with this rule     
+`startPort`<br/>*int*          | Minimum port that is matched by this rule             
+`endPort`<br/>*int*            | Maximum port that is matched by this rule             
+`etherType`<br/>*string*       | IP type, either IPv4 or IPv6              
+`protocol`<br/>*string*        | The IP protocol, either ICMP, TCP or UDP             
+`ingress`<br/>*boolean*        | Direction in which the rule is applied, false implies egress             
+`securityGroupId`<br/>*UUID*   | The ID of the parent security group             
+
+
+
+#### Create a security group rule
+
+```shell
+curl -X POST \
+    -H "MC-Api-Key: your_api_key" \
+    -H "Content-Type: application/json" \
+    -d "request_body" \
+    "https://api.your.cloudmc/v1/services/compute-os/devel/securitygrouprules"
+# Request should look like this:
+```
+```json
+{
+    "ingress": true,
+  	"protocol": "UDP",
+  	"startPort": 10,
+  	"endPort": 20,
+  	"remoteAddress": "0.0.0.0/0",
+    "securityGroupId": "f54f050b-01b2-4a73-b6e1-4e13a5566323",
+}
+```
+
+<code>POST /services/<a href="#service-connections">:service_code</a>/<a href="#environments">:environment_name</a>/securitygrouprules</code>
+
+Create a security group rule for a security group.
+
+ Required attributes          | Description                          
+ --------------------------   | ------------------------------------
+ `securityGroupId`<br/>*UUID* | The ID of the parent security group            
```

Comment:
They default to allow all IPs with any range. Should I make it clearer what the default values are for each field if not specified?